packages/sshnpdpy/lib/sshnpdclient.py

```diff
@@ -15,28 +16,35 @@
 
 from at_client import AtClient
 from at_client.common import AtSign
-from at_client.util import EncryptionUtil
+from at_client.util import EncryptionUtil, KeysUtil
 from at_client.common.keys import AtKey, Metadata, SharedKey
 from at_client.connections.notification.atevents import AtEvent, AtEventType
 
+from .sshrv import SSHRV
+
 
 class SSHNPDClient:
```

Comment:
calling this class a client is confusing

Reply:
https://github.com/atsign-foundation/sshnoports/pull/533/commits/de90028c6486f08eabdc39746ff85867fbcbbde9